process/banning.html

```diff
@@ -44,37 +44,48 @@ <h4>Nearby</h4>
 <ul>
 <li>from a single group, to the CEO;</li>
 <li>from a single group that the individual has not formally joined, to the chair(s) of the group; and</li>
 <li>from all W3C groups, to the CEO and COO jointly.</li>
 </ul>
 <p>If the chair, team contact, CEO, COO is the one accused of unacceptable behavior, they should not be involved in the decision to ban.</p>
-
-<p>2. Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, chairs may temporarily or permanently suspend participation by a person who has not formally joined the group.</p>
-
-<p>3. A W3C group chair and team contact may propose to suspend or remove a participant from a single group - typically for egregious and repetitive CEPC violations - by making a request to the CEO.</p>
-      <ul>
-        <li>If the group chair and team contact are not in agreement, each should explain their reasons.</li>
-        <li>If the participant is a W3C Member representative, the CEO will coordinate the response with the Member's Advisory Committee representative.</li>
-        <li>If anyone raises an issue to the CEO that a participant fails to meet the requirements of <a href="https://www.w3.org/Consortium/Process/#ParticipationCriteria">Individual Participation Criteria</a>, then the CEO is empowered to investigate. If the participant indeed is failing to meet those requirements, then after at least one warning and at least one subsequent violation, the CEO may temporarily or permanently suspend participation.</li>
-      </ul>
-<p>4. In extreme cases (such as death threats, or physical threats in general), the CEO and COO, acting together, may suspend or remove someone from participation in all groups. The CEO and COO must announce the action to W3C Management along with the rationale.</p>
-
-<p>5. Typically a suspension will encompass participation in all forms, including meetings, email, IRC, Github posting, etc. In no case may a person be blocked from seeing public information (e.g. mailing list or Github posts, public minutes).</p>
-
-<p>6. No individual may be banned from sending email to the W3C Ombuds team. (Bots may be banned.)</p>
-
-<p>7. The W3C Systems team may block spammers and others who abuse our systems.</p>
-
-<p>8. The W3C Systems team will provide technical support to implement suspensions.</p>
-
+</li>
+<li><p>In case of disagreement between the W3C group chair and team contact, each should explain their reasons to the CEO.</p></li>
+<li>
+  <p>Following violation of the <a href="https://www.w3.org/Consortium/cepc/">W3C Code of Ethics and Professional Conduct</a>, at least one warning by the chair(s), and at least one subsequent violation of the CEPC, a W3C group chair and team contact, acting together, may:</p>
```

Comment:
The CEPC's [Reporting](https://www.w3.org/Consortium/cepc/#Reporting) section appears to grant greater rights than this, saying:
> take such immediate action as they deem necessary 

Since the CEPC is mandated by the Process, I think we need to bring these guidelines into alignment with what the CEPC says.